src/pages/components/content-group-horizontal.mdx

```diff
@@ -11,15 +11,26 @@ import ComponentFeedback from 'components/ComponentFeedback';
 
 <AnchorLinks>
 
+<AnchorLink>Content guidance</AnchorLink>
 <AnchorLink>Design and functional specifications</AnchorLink>
 <AnchorLink>Development documentation</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)|
```

Comment:
As much as it pains me to say, the heading here is using Content block's heading. For what it's worth.

Reply:
Hahaha we will need to discuss this more ;)